test/TlsIconTest.php

```diff
@@ -0,0 +1,266 @@
+<?php
+
+namespace RouncubeTlsIcon\Tests;
+
+require_once __DIR__ . '/rcube_plugin.php';
+require_once __DIR__ . '/rcmail.php';
+
+use tls_icon;
+use rcmail;
+use PHPUnit\Framework\TestCase;
+
+final class TlsIconTest extends TestCase
+{
+
+	/** @var string */
+	private $strUnEnCrypted = '<img class="lock_icon" src="plugins/tls_icon/unlock.svg" title="Message received over an unencrypted connection!" />';
+
+	/** @var string */
+	private $strCryptedTlsv12 = '<img class="lock_icon" src="plugins/tls_icon/lock.svg" title="TLSv1.2" />';
+
+	/** @var string */
+	private $strCryptedTlsv12WithCipher = '<img class="lock_icon" src="plugins/tls_icon/lock.svg" title="TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits)" />';
+
+	/** @var string */
+	private $strInternal = '<img class="lock_icon" src="plugins/tls_icon/blue_lock.svg" title="Mail was internal" />';
+	public function testInstance()
```

Comment:
Too lazy to run a formatting linter right now, but is there intentionally no blank line here?

Reply:
Hmm, no, I pushed a line